pkgs/development/interpreters/php/default.nix

```
@@ -227,7 +227,7 @@ in {
   # We can't update the darwin versions because they simply don't compile at
   # all due to a bug in the intl extensions.
   #
-  # The bug so far is present in 7.1.21, 7.1.22, 7.1.23, 7.2.9, 7.2.10, 7.2.11.
+  # The bug so far is present in 7.1.21, 7.1.22, 7.1.23, 7.2.9, 7.2.10, 7.2.12.
```

```suggestion
  # The bug so far is present in 7.1.21, 7.1.22, 7.1.23, 7.2.9, 7.2.10, 7.2.11.
```
@ryantm can you accept this proposed change? Then it creates a new commit that needs to be squasched when merging.

---

I think we should add 7.2.12 to the list of broken versions and not just revert that line.